frontend/nuxt.config.js

```diff
@@ -351,13 +351,22 @@ export default {
     },
     manifest: {
       start_url: "/",
+      scope: "/",
       lang: "en",
+      dir: "auto",
       name: "Mealie",
       short_name: "Mealie",
+      id: "mealie",
```

Comment:
FWIW this defaults to the PWA's host at time of installation. Using a fixed value in the manifest will prevent users from installing potentially multiple different Mealie instances. Not that this is a common use case but worth mentioning nontheless.

https://developer.chrome.com/docs/capabilities/pwa-manifest-id